src/containers/Tablet/Tablet.tsx

```diff
@@ -0,0 +1,113 @@
+import {useCallback, useEffect, useRef} from 'react';
+import {useParams} from 'react-router';
+import {useDispatch} from 'react-redux';
+import cn from 'bem-cn-lite';
+import {Link as ExternalLink} from '@gravity-ui/uikit';
+
+import {backend} from '../../store';
+import {getTablet, getTabletDescribe} from '../../store/reducers/tablet';
+import {useAutofetcher, useTypedSelector} from '../../utils/hooks';
+import '../../services/api';
+
+import EntityStatus from '../../components/EntityStatus/EntityStatus';
+import {Tag} from '../../components/Tag';
+import {Icon} from '../../components/Icon';
+import {EmptyState} from '../../components/EmptyState';
+import {Loader} from '../../components/Loader';
+
+import {TabletTable} from './TabletTable';
+import {TabletInfo} from './TabletInfo';
+import {TabletControls} from './TabletControls';
+
+import './Tablet.scss';
+
+export const b = cn('tablet-page');
+
+export const Tablet = () => {
+    const isFirstDataFetchRef = useRef(true);
+
+    const dispatch = useDispatch();
+
+    const params = useParams<{id: string}>();
+    const {id} = params;
+
+    const {
+        data: tablet = {},
+        loading,
+        id: tabletId,
+        history = [],
+        tenantPath,
+    } = useTypedSelector((state) => state.tablet);
+
+    useEffect(() => {
+        if (isFirstDataFetchRef.current && tablet && tablet.TenantId) {
+            dispatch(getTabletDescribe(tablet.TenantId));
+            isFirstDataFetchRef.current = false;
+        }
+    }, [dispatch, tablet]);
+
+    const fethcData = useCallback(() => {
+        dispatch(getTablet(id));
```

Comment:
there was a version validity check. Is it not needed?

Reply:
Currently actual ydb versions are stable-22-* and  stable-23-* or stable-32. So I consider these check excessive. And, I think, it will be better, if we will return some response error, but not null. However, thanks to your comment, I discovered that i didn't add any error check